pkg/pillar/types/zedroutertypes.go

```diff
@@ -323,10 +323,15 @@ type CryptoBlock struct {
 type WifiConfig struct {
 	SSID      string            // wifi SSID
 	KeyScheme WifiKeySchemeType // such as WPA-PSK, WPA-EAP
-	Identity  string            // identity or username for WPA-EAP
-	Password  string            // string of pass phrase or password hash
-	Crypto    CryptoBlock       // encrypted block of items
-	Priority  int32
+	// to be deprecated, use CipherBlock instead
+	Identity string // identity or username for WPA-EAP
+	// to be deprecated, use CipherBlock instead
+	Password string      // string of pass phrase or password hash
+	Crypto   CryptoBlock // encrypted block of items
```

Comment:
CryptoBlock should presumably be expunged completely since you have defined CipherBlock for the same purpose

Reply:
will do.